tests/test_blending.py

```diff
@@ -234,3 +234,41 @@ def test_softmax_rgb_blend(self):
 
         self.assertTrue(torch.allclose(dists1.grad, dists2.grad, atol=2e-5))
         self.assertTrue(torch.allclose(zbuf1.grad, zbuf2.grad, atol=2e-5))
+
+    def test_blend_params(self):
+        """Test colour parameter of BlendParams().
+        Assert default value is equal to same passed value (1.0, 1.0, 1.0).
+        Assert colour setting propagates through the renderer to output image
+        by comparing output pixel at 0, 0 with background colour.
+        """
+        bp1, bp2 = BlendParams(), BlendParams(background_color=(1.0, 1.0, 1.0))
+        self.assertAlmostEqual(bp1.background_color, bp2.background_color)
+
+        background_color = torch.tensor((0.5, 0.5, 0.5))
+        bp3 = BlendParams(background_color=background_color)
```

Comment:
I think this is enough for the specific change you are adding - setting the type of `Sequence` allows the value of `background_color` to be set on initialization.  

We have a bigger task for enabling background colors for rasterization which will affect all the blending functions so you don't need to test this.